labonneboite/scripts/update_metiers_tension.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+
+# get csv file from : https://statistiques.pole-emploi.org/bmo/bmo?in=4&lg=0&pp=2019&ss=1#
+# To build a new file, we join the old file with the new one and we update the pourcentage de tension
```

Comment:
But alors you are french? :-D

https://www.youtube.com/watch?v=s8cMrDqwbNw

Reply:
Fan ultime de Louis de Funes, touché !